sample/src/main/java/io/getstream/chat/example/ChannelMoreActionDialog.java

```diff
@@ -0,0 +1,106 @@
+package io.getstream.chat.example;
+
+import android.app.Activity;
+import android.app.Dialog;
+import android.content.Context;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.WindowManager;
+import android.widget.LinearLayout;
+
+import androidx.annotation.NonNull;
+
+import com.getstream.sdk.chat.R;
+import com.getstream.sdk.chat.model.Channel;
+import com.getstream.sdk.chat.utils.ResultCallback;
+import com.getstream.sdk.chat.utils.Utils;
+import com.getstream.sdk.chat.viewmodel.ChannelListViewModel;
+
+public class ChannelMoreActionDialog extends Dialog {
+
+    Channel channel;
+    ChannelListViewModel viewModel;
+    Context context;
```

Comment:
Yeah, exactly. But we can't use getContext() in here because `ChannelMoreActionDialog` has different `Theme`.

```java
    public ChannelMoreActionDialog(@NonNull Context context) {        
        super(context, R.style.DialogTheme);
        ...
    }
```
When showing Toast, it looks pretty ugly(Toast layout would be messed up).
So we must use the `Context` of `MainActivity`